common/build/eslint-config-fluid/src/custom-rules/no-release-tags.js

```diff
@@ -0,0 +1,54 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+const { TSDocParser } = require("@microsoft/tsdoc");
+
+function hasReleaseTag(comment) {
+	const parser = new TSDocParser();
+	const parserContext = parser.parseString(comment);
+
+	const hasReleaseTag =
+		parserContext.docComment.modifierTagSet.isAlpha() ||
+		parserContext.docComment.modifierTagSet.isBeta() ||
+		parserContext.docComment.modifierTagSet.isPublic() ||
+		parserContext.docComment.modifierTagSet.isInternal();
+
+	return hasReleaseTag;
+}
+
+module.exports = {
+	meta: {
+		type: "problem",
+		docs: {
+			description: "This rule restricts any release tags on member class and interface.",
+			category: "Best Practices",
+		},
+		schema: [],
+		messages: {
+			releaseTagOnMember: "Release tag on member class / interface found.",
+		},
+	},
+	create(context) {
+		return {
+			ClassDeclaration(node) {
```

Comment:
Remember, we need to target more than just `ClassDeclaration`s. We need to target classes, interfaces, and types. I don't know which TypeScript AST concepts we need to target for that. For classes, I expect we need both `ClassDeclaration`s and `ClassExpression`s. For the others, I don't know for sure which matchers we need.